spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/config/WebsocketProperties.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.gateway.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.util.unit.DataSize;
+
+/**
+ * Configuration properties for the websoket.
+ */
+@ConfigurationProperties("spring.cloud.gateway.websocket")
+public class WebsocketProperties {
+
+	/** The max frame payload length, the default is 65536 (64KB). */
+	private DataSize maxFramePayloadLength;
```

Comment:
There already exists a property here https://github.com/spring-cloud/spring-cloud-gateway/blob/master/spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/config/HttpClientProperties.java#L567 I think it could be reused.